src/components/LoanFinding/LendingCategorySection.vue

```diff
@@ -125,10 +133,39 @@ export default {
 			}
 			return '336px';
 		},
+		multipleAmount() {
+			let amount = 0;
+			for (let index = 0; index < this.totalLoans; index += 1) {
+				const loan = this.loans[index];
+				const { unreservedAmount } = loan;
+
+				if (this.enableFiveDollarsNotes) {
+					amount += this.userBalance > 20 ? 25 : 5;
```

Comment:
The `unreservedAmount` needs to be checked here to, like in the `else`

Reply:
TY!